daemon/api.go

```diff
@@ -1568,7 +1568,8 @@ out:
 		if err != nil {
 			return BadRequest("cannot read snap file: %v", err)
 		}
-		snapName = info.Name()
+		snapName = info.InstanceName()
+		// TODO parallel-install: use proper name
```

Comment:
Why is this adding those TODOs instead of simply having the StoreName function and using it already?  Would be shorter and more clear, I suspect. Then, TODO messages would only be used for things which aren't that trivial, if any.

Reply:
so introduce a StoreName that does the same as InstanceName for now/in this branch? that would work for me

Reply:
It's fine either way. A different PR might be better given that we've already had several rounds on it. Otherwise everybody needs to come back.